js/jquery.mapael.js

```diff
@@ -219,6 +219,11 @@
                 self.onUpdateEvent(e, opt);
             });
 
+            // Attach showElementsInRange event
+            self.$container.on("showElementsInRange." + pluginName, function (e, ranges) {
```

Comment:
Does  the argument `ranges` shouldn't be called `options` because we can set options like hiddenOpacity or animDuration in this object ?